brave/src/main/java/com/linecorp/armeria/common/brave/BraveBlockHoundIntegration.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.brave;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+
+import reactor.blockhound.BlockHound.Builder;
+import reactor.blockhound.integration.BlockHoundIntegration;
+
+/**
+ * A {@link BlockHoundIntegration} for the brave module.
+ */
+@UnstableApi
+public final class BraveBlockHoundIntegration implements BlockHoundIntegration {
```

Comment:
Is there any case a user needs to refer to our `BlockHoundIntegration` implementations manually? If not, can we move all `BlockHoundIntegration` into the `internal` package, given that they are loaded automatically via SPI?

Reply:
I thought users may want to exclude some `BlockHoundIntegration`s.

e.g. 
> BlockHound.install(BlockHoundIntegration... integrations) - same as BlockHound.install(), but adds user-provided integrations to the list.
BlockHound.builder().install() - will create a new builder, without discovering any integrations.
You may install them manually by using BlockHound.builder().with(new MyIntegration()).install().

https://github.com/reactor/BlockHound/blob/master/docs/customization.md#customization

If there is no need, I'm fine with hiding them. Let me know what you think

Reply:
I see. Then it's fine as it is :-)